backend/main.go

```diff
@@ -91,6 +91,23 @@ func (w *Worker) processIncomingEthereumSignatureRequest(sr store.SignatureReque
 		return errors.New("transaction withdraw time expired")
 	}
```

Comment:
Well, this comment does not apply is we have a single worker.

Reply:
Yes, we have just one worker. Also, processing all ledgers up to latest one before handling signing requests ensures that all new transactions will have their time bounds calculated using the latest ledger close time.

Reply:
I think we are safe on the backend side because it is responsible for processing only a valid requests (i.e. which has a corresponding Ethereum deposit transaction). I'm not up to date with recent stats but Google says it's 30 tx/sec. This is definitely something a single instance can process.

In Stellar, it's 200 tx/sec. It's much higher than Ethereum but I still think it's something a single instance should handle. The number also assumes that all ops in a ledger will be depositing to a single Starbridge cluster.

Regarding frontend instances - they should be stateless so operators should be able to deploy multiple of them.

@tomerweller can you double check/confirm requests/second and deposits/second assumptions above?